cmd/thor/flags.go

```diff
@@ -151,6 +151,12 @@ var (
 		Usage: "metrics service listening address",
 	}
 
+	allowedTracersFlag = cli.StringFlag{
+		Name:  "allowed-tracers",
```

Comment:
```suggestion
		Name:  "api-allowed-tracers",
```

@otherview to keep consistency with `api-allow-custom-tracers`

Reply:
Also, update the usage.md if you can

Reply:
Good point - fixed !